app/vue/src/client/preview/render.js

```diff
@@ -67,6 +67,8 @@ export function renderMain(data, storyStore) {
     //    https://github.com/storybooks/react-storybook/issues/81
     previousKind = selectedKind;
     previousStory = selectedStory;
+  } else {
```

Comment:
Is this not also a valid change for react? totally guessing here.

either way this PR is good to merge, as it addresses a very nasty bug for Vue users!

Reply:
yup, In the Vue, occure the similar issues.